tanzu-build-service/descriptors.md

```diff
@@ -3,14 +3,11 @@ title: Descriptors
 owner: Build Service Team
 ---
 
-This topic describes the descriptors that are available so you can choose
-which option to configure depending on your use case.
-
 ## <a id="descriptors"></a> About descriptors
 
 Tanzu Build Service descriptors are curated sets of dependencies, including stacks and buildpacks, that are
-continuously released on VMware Tanzu Network to resolve all workload Critical and High CVEs.
-Descriptors are imported into Tanzu Build Service to update the entire cluster.
+continuously released on VMware Tanzu Network to resolve all workload Critical and High CVEs. Descriptors are only used 
+if TAP is configured for automatic dependency updates. Descriptors are imported into Tanzu Build Service to update the entire cluster.
```

Comment:
Not sure if this is the right place for it.  But we should call out that new buildpacks and stacks are shipped in patch TAP versions.

Reply:
good call